### PR TITLE
New Lint quick-fix:  unnecessary_override

### PR DESCRIPTION
**Pull request by JPaulsen**
From [Linter](https://github.com/dart-lang/linter/pull/614)

### CODE REVIEW

File: pkg/analysis_server/test/services/correction/fix_test.dart [hunk 1]
```diff
@@ -5986,6 +5986,74 @@ var x;
...
+class B extends A {
+
+}
+''');
+  }
+
+  test_removeMethodDeclaration_method() async {
+    String src = '''
+class A {
+@override
```
Comment (JPaulsen) [inline pkg/analysis_server/test/services/correction/fix_test.dart]: @bwilkerson is there a way to safely remove the indentation and the new line when removing the method?

Comment (bwilkerson) [inline pkg/analysis_server/test/services/correction/fix_test.dart]: I think that `utils.getLinesRange` will do what you're looking for.

Comment (JPaulsen) [inline pkg/analysis_server/test/services/correction/fix_test.dart]: It worked!, Fixed

**Review by bwilkerson — APPROVED** [review]
It would be good if one of the existing tests were extended to have members before and after the one being removed.

**Pull request merged**